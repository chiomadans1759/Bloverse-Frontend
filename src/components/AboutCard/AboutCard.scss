.about-body {
  width: 60%;
  margin: 50px auto;
}

@media screen and (max-width: 768px) {
.about-body{
  width:90%
  }
}

@media screen and (max-width: 600px) {
.about-body {
  width:90%
  }
}

@media screen and (max-width: 360px) {
.about-body {
  width:90%
  }
}
