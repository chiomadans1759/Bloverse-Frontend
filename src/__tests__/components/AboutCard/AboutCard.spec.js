import { mount } from '@vue/test-utils'
import AboutCard from '@/components/AboutCard/AboutCard.vue';

describe('AboutCard Compoenent', () => {
  it('should render correctly', () => {

    const wrapper = mount(AboutCard, {});
    expect(wrapper.element).toMatchSnapshot();
  });
});
