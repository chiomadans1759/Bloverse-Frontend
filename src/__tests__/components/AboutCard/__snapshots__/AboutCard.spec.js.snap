// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutCard Compoenent should render correctly 1`] = `
<div>
  <div
    class="about-body"
  >
    <div />
  </div>
</div>
`;
